### ISSUE TITLE
Make lable compatible with existing ones

### ISSUE THREAD

**Issue by Mic92** [timokau/marvin-mk2#43]
Our current labels start with number for priority and also have a color scheme according to priority: https://github.com/NixOS/nixpkgs/pull/92249
Marvin should also follow the same scheme.

**Comment by blaggacao** [comment]
I'd wait for GA version of marvin, since "looking official" implies stronger stability guarantees.

**Comment by timokau** [comment]
I don't think it should follow that scheme. There is not much to be gained, and the priority number is a major inconvenience when handling those labels. That is especially true for labels that we want people to actively search for. They would now have to remember/look up the somewhat arbitrary number prefix and probably quote the label in the search. Additionally its nice to have a 1:1 match between the labels and the status names.

(Also, not all current nixpkgs label match that scheme.)

Colors could be added of course, I don't mind that.
